department-of-veterans-affairs/va.gov-cms
# VACMS-3790: Facility status permissions, and some permissions cleanup

## Description

See #2063 (update the Facility Status dashboard) 

and #3790 (update permissions for facility nodes)

As part of 3790, i also undertook a cleanup of existing *node* module permissions for three editorial workflow roles (editor/reviewer/publisher). This PR:
* adds "edit any [content type]" permissions to some new content types (`basic_landing_page`, `campaign_landing_page`, etc). it's not truly _any_, because access to edit is determined by workbench access settings. This was always the intent, but some new content types have missed this. 
  * removes all "edit own [content type]" permissions, which are redundant, because all roles already have "edit any _content type_".  
* Standardizes "view revisions" for all content types for all users (except `CMS user guide` content type). Always the intent, but some new content types were missing this.
* Standardizes "revert revisions" for all content types for content_publisher (still requires workbench_access). Always the intent, but some new content types were missing this.

## Testing done
Visual, phpunit. 


## QA steps

As user sandy.tadeo@va.gov (content_publisher, VBA)
1. Go to /admin/content
2. You will see a Facility status tab (previously, this was one level up in the admin menu, and only visible to content_admins)
3. you should be able to click on it and see all facility statuses 
4. you should only be able to edit those that are owned by VBA. 
5. As content publisher, you should be able to use bulk edit to publish latest revision

As user jonathan.wehausen@va.gov (content editor, all sections) 
1. Go to /admin/content
2. You will see a Facility status tab
3. you should be able to click on it and see all facility statuses
4. you should only be able to edit any facility. 
5. As content editor, you should not be able to use bulk edit 


### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
